### PR TITLE
ID:FPCO-3617 ; DONE: 90; HOURS: 2; FDK theme package command that will generate zip file of theme

### PR DESCRIPTION
**Pull request by anuragpandey1115**

### CODE REVIEW

File: src/lib/Theme.ts [hunk 1]
```diff
@@ -913,4 +913,19 @@ export default class Theme {
...
+            let filepath = path.join(process.cwd(), 'template', 'package.json');
+            let packageContent: any = readFile(filepath);
+            let Content = JSON.parse(packageContent);
+            await archiveFolder({
+                srcFolder: path.join(process.cwd(),'template'),
+                destFolder: path.join(process.cwd()),
+                zipFileName: `${Content.name}_${Content.version}.zip`,
+            });
+        } catch (err) {
+            throw new CommandError(`Failed to copying theme files to .fdk folder`);
```
Comment (vivek8690) [inline src/lib/Theme.ts]: change this error message @anuragpandey1115

**Review by vivek8690 — CHANGES_REQUESTED** [review]
Add proper error message

File: src/lib/Theme.ts [hunk 2]
```diff
@@ -913,4 +913,19 @@ export default class Theme {
             throw new CommandError(err.message, err.code);
         }
     };
+
+    public static generateThemeZip = async () => {
+        let filepath = path.join(process.cwd(), 'template', 'package.json');
+        let packageContent: any = readFile(filepath);
+        let Content = JSON.parse(packageContent);
```
Comment (vivek8690) [inline src/lib/Theme.ts]: local variables should start with a small letter @anuragpandey1115

**Review by vivek8690 — COMMENTED** [review]
variable name should start with small letter

**Comment by Shivraj0** [comment]
https://github.com/gofynd/fdk-cli/pull/131/

**Pull request closed without merging**